### DISCUSSION TITLE
In need of usable payoff function

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#81] [people; relating]
SD model has high correlation (collinearity) and with hierarchical structure being added, is loglikelihood for stocked rate series usable payoff function? Gaussian process introduced in birthday casestudy (chp.24 of BDA) seems to the best choice, which models time-series' covariance. Mike's hierarchical gaussian process is a good starting point. 

Last week, Tamara suggested: https://research.ibm.com/publications/are-you-using-test-log-likelihoods-correctly

All in all, a sbc structure that addresses model structure is in need as described in https://github.com/hyunjimoon/SBC/issues/41 e.g. sbc in gaussian process, implicit, hierarchical model

**Comment by tomfid** [comment]
Corr in parameters or outputs? Over time or across hierarchy? One likely source would be lack of variation in initial conditions.

**Reply by hyunjimoon** [reply]
Good point.

Realistic reason, outcome correlation is unavoidable.

Computational efficiency reason, parameter correlation between P estimated parameter is needed.

This is why visual checks is essential.

https://discourse.mc-stan.org/t/best-practices-for-simulation-based-calibration-with-hierarchical-models/19630/3

in pin this context

**Comment by hyunjimoon** [comment]
Classify SBC as payoff function: after defining the SBC loss function we can even compute its gradient.
